### PR TITLE
thriftbp: Change the return type of Merge to BaseplateProcessor

### PR DESCRIPTION
**Pull request by fishy**

### CODE REVIEW

File: thriftbp/merger.go [hunk 1]
```diff
@@ -1,17 +1,15 @@
...
-import (
-	"github.com/apache/thrift/lib/go/thrift"
-)
-
-// Merge merges together the specified processors.
+// Merge merges together multiple processors into the first one.
 //
-// It's useful if the server need to support two separated thrift files.
+// It's useful when the server need to support more than one separated thrift
+// files.
```
Comment (pacejackson) [inline thriftbp/merger.go:6]: I think this should be “file”?

Comment (fishy) [reply thriftbp/merger.go:6]: `more than one` so it should be plural?

Comment (pacejackson) [reply thriftbp/merger.go:6]: What about “more than one thrift file”?

Comment (fishy) [reply thriftbp/merger.go:6]: oh so you are thinking about "more than (one thrift file)" and I'm thinking about "(more than one) thrift files".

Comment (fishy) [reply thriftbp/merger.go]: done.

**Pull request merged**